frontend/src/models/propertyDefinitionsModel.ts

```diff
@@ -25,7 +25,7 @@ export const propertyDefinitionsModel = kea<
             {
                 loadPropertyDefinitions: async (initial?: boolean) => {
                     const url = initial
-                        ? 'api/projects/@current/property_definitions/?limit=5000'
+                        ? 'api/projects/@current/property_definitions/?limit=500'
```

Comment:
Hmm. Is this change needed. Not sure what the implications here are.


Originally w e added this since netdata has a bunch of custom properties. and loading them all took forwever. However with new properties filter in insights not sure if we need this :thinking:

Reply:
Hmm yeah you have a point here, if this works for the filters it should be fine for the property definitions page too. I undid this change, let's see if just paginating the frontend for it will take care of the freezing. (Though I did encounter the API call itself crashing when I went to the property definitions page)